source/Patches/RainbowMod/PalettePatch.cs

```diff
@@ -8,31 +8,33 @@ public static void Load()
         {
             var array = new[]
```

Comment:
Maybe rename these array variables to something that actually describes it.

Reply:
opted to not use a variable since its only used once